manifests/m/Microsoft/OneDrive/Internal/Fast/21.246.1129.0001/Microsoft.OneDrive.Internal.Fast.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OneDrive.Internal.Fast
+PackageVersion: 21.246.1129.0001
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent /update /restart
+  SilentWithProgress: /silent /update /restart
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitFast/21.246.1129.0001/amd64/OneDriveSetup.exe
+  InstallerSha256: FE3F7B2FCA5562803CF3C8829459D7B7D7F824BDB2634B704AFB9D43FE34DDAD
+- Architecture: x64
```

Seems like there is a bug somewhere
```suggestion
- Architecture: x86
```
Unless you want to do Custom: /allusers on this one

---

I think this bug occurred after installer detection improvements was merged. What I think (still not confirmed since there might be a bug 🐛 in automation also) is the logic implemented by **Trenly** copies complete installer instead of just hash and that's where architecture and other properties are copied, hence duplicate installer entry. 

Reference: https://github.com/microsoft/winget-pkgs/blob/master/Tools/YamlCreate.ps1#L420

---

Just checked https://github.com/microsoft/winget-pkgs/blob/525fdb9b543a131b7dc375410e318f38b6eec311/manifests/m/Microsoft/OneDrive/Internal/Fast/21.239.1118.0001/Microsoft.OneDrive.Internal.Fast.installer.yaml broke the manifest